kommunicate/src/main/java/com/applozic/mobicomkit/broadcast/AlMessageEvent.java

```diff
@@ -130,5 +130,6 @@ public static class ActionType extends JsonMarker {
         public static final String USER_ACTIVATED = "USER_ACTIVATED";
         public static final String USER_DEACTIVATED = "USER_DEACTIVATED";
         public static final String AWAY_STATUS = "AWAY_STATUS";
+        public static final String ACTION_TEXT_UPDATED = "ACTION_TEXT_UPDATED";
```

Comment:
The name is very vague and does not tell what exactly this Broadcast does, change it to suit the feature better.